src/SPIN/compute_spin.cpp

```diff
@@ -217,8 +217,8 @@ void ComputeSpin::compute_vector()
   magtot[2] *= scale;
   magtot[3] = sqrt((magtot[0]*magtot[0])+(magtot[1]*magtot[1])+(magtot[2]*magtot[2]));
   spintemperature = hbar*tempnumtot;
-  // spintemperature /= (2.0*kb*tempdenomtot);
-  spintemperature /= (kb*tempdenomtot);
+  spintemperature /= (2.0*kb*tempdenomtot);
+  // spintemperature /= (kb*tempdenomtot);
```

Comment:
Could you please remove the commented out line? If anybody ever reads this code, it is really confusing.

Reply:
Sure.